### DISCUSSION TITLE
Import recipe from HelloFresh

### DISCUSSION THREAD

**Discussion by kevinholtkamp** [mealie-recipes/mealie#2786] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Hello Fresh is a very nice tool to try out new recipes, without having to buy full amounts of the ingredients.
It would be nice to use Hello Fresh to try out new recipes, and if you like them, add them to mealie and continue using them.

### Please provide a concise description of the feature that would resolve your issue.

Importing recipes from Hello Fresh into mealie 

### Please consider and list out some caveats or tradeoffs made in your design decision

/

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
This should already be supported:

https://github.com/hhursev/recipe-scrapers

Have you tried importing hello fresh recipes via URL?

**Comment by Kuchenpirat** [comment]
yep, this totally works. 

Just tried it with:
https://www.hellofresh.com/recipes/when-steak-met-potatoes-5857fcd16121bb11c124f383

**Comment by kevinholtkamp** [comment]
I'm so sorry guys, I totally misunderstood the documentation [here](https://nightly.mealie.io/documentation/getting-started/features/#:~:text=Mealie%20supports%20importing%20recipes%20from%20a%20few%20other%20sources%20besides%20websites.%20Currently%20the%20following%20sources%20are%20supported%3A).

On that topic, maybe the webscraping should be elaborated on a little bit more in there?

**Reply by boc-the-git** [reply]
PRs to improve doco are always welcome
